src/users/users.controller.ts

```diff
@@ -173,20 +180,44 @@ export class UsersController {
           user.password,
         ),
       );
+      return newUser;
+    } catch ({ response = 'Something went wrong', status = 500 }) {
+      throw new HttpException(response, status)
+    }
```

Comment:
Use the `message` field instead of `response`. Avoid using destructuring assignment in `catch` statement since `err` has unknown or `any` type depending on tsc settings
 
```suggestion
    } catch (err) {
      throw new HttpException(err.message ?? 'Something went wrong', err.status ?? 500)
    }
```